azure-iot-hub/azure/iot/hub/digital_twin_service_client.py

```diff
@@ -0,0 +1,44 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from .auth import ConnectionStringAuthentication
+
+from .protocol.iot_hub_gateway_service_ap_is20190701_preview import (
+    IotHubGatewayServiceAPIs20190701Preview as pl_client,
+)
+from .protocol.models.digital_twin_interfaces import DigitalTwinInterfaces
+from .protocol.models.digital_twin_interfaces_patch import DigitalTwinInterfacesPatch
+
+DigitalTwin = DigitalTwinInterfaces
+DigitalTwinPatch = DigitalTwinInterfacesPatch
+
+
+class DigitalTwinServiceClient(object):
+    def __init__(self, connection_string):
+        self.auth = ConnectionStringAuthentication(connection_string)
+        self.pl = pl_client(self.auth, "https://" + self.auth["HostName"])
+
+    def get_digital_twin(self, digital_twin_id):
+        return self.pl.digital_twin.get_interfaces(digital_twin_id)
+
+    def get_digital_twin_interface_instance(self, digital_twin_id, interface_instance_name):
```

Comment:
Can we get docs for these? I'm not clear on what some of this stuff is supposed to do.